src/server/game/Battlegrounds/Zones/BattlegroundAB.h

```diff
@@ -212,11 +212,11 @@ const uint32 BG_AB_GraveyardIds[BG_AB_ALL_NODES_COUNT] = {895, 894, 893, 897, 89
 // x, y, z, o
 const float BG_AB_BuffPositions[BG_AB_DYNAMIC_NODES_COUNT][4] =
 {
-    {1185.71f, 1185.24f, -56.36f, 2.56f},                   // stables
-    {990.75f, 1008.18f, -42.60f, 2.43f},                    // blacksmith
-    {817.66f, 843.34f, -56.54f, 3.01f},                     // farm
-    {807.46f, 1189.16f, 11.92f, 5.44f},                     // lumber mill
-    {1146.62f, 816.94f, -98.49f, 6.14f}                     // gold mine
+    {1185.566f, 1184.629f, -56.36329f, 2.303831f},         // stables
+    {989.9393f, 1008.75f, -42.60327f, 0.8203033f},         // blacksmith
```

Comment:
This difer 
https://github.com/TrinityCore/TrinityCore/commit/de4986d7d406120d5fd89bada3e06827e30fa604#diff-fa90798e35e64a007eb50d9c6385dee0R80 maybe it changed on newer versions or something else?

Reply:
3.3.5a sniff:

[40] GUID: Full: Type: GameObject Entry: 180148 Low: 2302
[40] Object Type: 5 (GameObject)
[40] Update Flags: 848 (LowGuid, StationaryObject, GOPosition, GORotation)
[40] GO Transport GUID: 0x0
[40] GO Position: X: 990.1131 Y: 1008.73 Z: -42.60328
[40] GO Orientation: 0.8203033